libs/sdk-ui-ext/src/dashboardView/DashboardWidgetRenderer/InsightRenderer/InsightRenderer.tsx

```diff
@@ -218,33 +220,52 @@ export const InsightRenderer: React.FC<IInsightRendererProps> = ({
      */
     const drillableItemsToUse = onDrill ? drillableItems ?? implicitDrills : undefined;
     return (
-        <IntlWrapper locale={locale}>
-            {(status === "loading" || status === "pending" || isVisualizationLoading) && <LoadingComponent />}
-            {(error || visualizationError) && (
-                <InsightError
-                    error={error || visualizationError}
-                    ErrorComponent={ErrorComponent}
-                    height={null} // make sure the error is aligned to the top (this is the behavior in gdc-dashboards)
-                />
-            )}
-            {status === "success" && (
-                <InsightRendererImpl
-                    insight={insightWithAddedWidgetProperties}
-                    backend={effectiveBackend}
-                    workspace={effectiveWorkspace}
-                    drillableItems={drillableItemsToUse}
-                    onDrill={onDrill ? handleDrill : undefined}
-                    config={chartConfig}
-                    onLoadingChanged={handleLoadingChanged}
-                    locale={dashboardViewConfig.locale ?? (userWorkspaceSettings.locale as ILocale)}
-                    settings={userWorkspaceSettings}
-                    colorPalette={colorPalette}
-                    onError={handleError}
-                    pushData={handlePushData}
-                    ErrorComponent={ErrorComponent}
-                    LoadingComponent={LoadingComponent}
-                />
-            )}
-        </IntlWrapper>
+        <div
+            style={{
```

Comment:
Please extract this to a constant outside of the component and use it here. This will improve performance.